lib/plugins/aws/package/compile/events/eventBridge/index.test.js

```diff
@@ -1334,3 +1336,121 @@ describe('AwsCompileEventBridgeEvents', () => {
     });
   });
 });
+
+describe('EventBridgeEvents', () => {
+  after(fixtures.cleanup);
+
+  describe('when using the default event bus arn', () => {
+    let cfResources;
+    let naming;
+
+    before(() =>
+      fixtures
+        .extend('function', {
+          functions: {
+            foo: {
+              events: [
+                {
+                  eventBridge: {
+                    eventBus: 'arn:aws:events:us-east-1:12345:event-bus/default',
+                    schedule: 'rate(10 minutes)',
+                  },
+                },
+              ],
+            },
+          },
+        })
+        .then(fixturePath =>
+          runServerless({
+            cwd: fixturePath,
+            cliArgs: ['package'],
+          })
+        )
+        .then(({ cfTemplate, awsNaming }) => {
+          ({ Resources: cfResources } = cfTemplate);
+          naming = awsNaming;
+        })
+    );
+
+    it('should create the correct execution role', () => {
+      const executionRolePolicies =
+        cfResources.IamRoleCustomResourcesLambdaExecution.Properties.Policies;
+      expect(executionRolePolicies[0].PolicyDocument.Statement).to.deep.equal([
+        {
+          Action: [
+            'events:PutRule',
+            'events:RemoveTargets',
+            'events:PutTargets',
+            'events:DeleteRule',
+          ],
+          Effect: 'Allow',
+          Resource: {
+            'Fn::Join': [
+              ':',
+              [
+                'arn',
+                { Ref: 'AWS::Partition' },
+                'events',
+                {
+                  Ref: 'AWS::Region',
+                },
+                {
+                  Ref: 'AWS::AccountId',
+                },
+                'rule/service-dev-foo-rule-1',
+              ],
+            ],
+          },
+        },
+        {
+          Action: ['lambda:AddPermission', 'lambda:RemovePermission'],
+          Effect: 'Allow',
+          Resource: {
+            'Fn::Join': [
+              ':',
+              [
+                'arn',
+                { Ref: 'AWS::Partition' },
+                'lambda',
+                {
+                  Ref: 'AWS::Region',
+                },
+                {
+                  Ref: 'AWS::AccountId',
+                },
+                'function',
+                'service-dev-foo',
+              ],
+            ],
+          },
+        },
+      ]);
```

Comment:
Let's maybe not confirm on whole structure (it's problematic maintenance wise) but instead simply test that there's a policy which allows `'events:PutRule'` on rule of which ARN ends with `rule/service-dev-foo-rule-1`